openmrs-19/src/main/java/org/motechproject/openmrs19/tasks/impl/OpenMRSActionProxyServiceImpl.java

```diff
@@ -146,6 +147,27 @@ public void createProgramEnrollment(String configName, String patientUuid, Strin
         programEnrollmentService.createProgramEnrollment(configName, programEnrollment);
     }
 
+    @Override
+    public void changeStateOfProgramEnrollment(String configName, String programEnrollmentUuid, DateTime programCompletedDate,
+                                               String stateUuid, DateTime startDate) {
+        ProgramEnrollment programEnrollment = new ProgramEnrollment();
+        programEnrollment.setUuid(programEnrollmentUuid);
+        programEnrollment.setDateCompleted(Objects.nonNull(programCompletedDate) ? programCompletedDate.toDate() : null);
+
+        if (!Strings.isNullOrEmpty(stateUuid)) {
```

Comment:
we are usually using StringUtils from apache.commons, this class is already imported here so I think you should use this